tcl/is.tcl

```diff
@@ -534,10 +266,58 @@ namespace eval qc::is {
         return [expr {[string toupper $bool] in {Y N YES NO TRUE FALSE T F 0 1}}]
     }
 
-    proc decimal {number} {
+    proc decimal {args} {
```

Comment:
Should we raise an error if developer specifies scale but not precision ?

My first impression from tcl tests and examples was but precision and scale were both optional and not dependent on each other.